### ISSUE TITLE
PSW location options being read aloud as "blank"

### ISSUE THREAD

**Issue by CorinaMala** [visitscotland/dot-com-brxm#336]
### Raised by
Corina Malanca

### Describe the bug
Given the someone uses a screen reader (NVDA)
And they access the PSW on any page
When the user tabs through the location options
Then the screen reader reads the options as "blank"

However, the ChromeVox screen reader reads the dropdown options correctly.

### To Reproduce
1. Go to 'https://www.visitscotland.com/'
2. Access NVDA
3. Scroll down to 'PSW'
4. Open the location dropdown and tab through the options
5. Listen to the error

### Expected behavior
NVDA should be able to voice the location options as ChromeVox does.

### Page URL
All pages that contain PSW

### Environment
All

### Browser
All

### Device and version
If applicable, device the bug was seen on
